db/migrate/20161003145646_add_position_to_links.rb

```diff
@@ -0,0 +1,5 @@
+class AddPositionToLinks < ActiveRecord::Migration
+  def change
+    add_column :links, :position, :integer, null: false, default: 0
```

Comment:
ha oh dear - that was how it was wrote originally before I intervened.

Thanks @boffbowsh 👍  - Sorry @rubenarakelyan